cloudify_gcp/utils.py

```diff
@@ -170,11 +170,30 @@ def create(resource):
     return resource.create()
 
 
+def runtime_properties_cleanup(ctx):
+    # cleanup runtime properties
+    # need to convert generaton to list, python 3
+    keys = [key for key in ctx.instance.runtime_properties.keys()]
+    for key in keys:
+        del ctx.instance.runtime_properties[key]
+
+
 def delete_if_not_external(resource):
-    if not should_use_external_resource():
+    if not should_use_external_resource(ctx):
         return resource.delete()
     else:
-        ctx.instance.runtime_properties.pop(constants.RESOURCE_ID, None)
+        runtime_properties_cleanup(ctx)
+
+
+def resorce_created(ctx, resource_field):
```

Comment:
`resource_created` (with ou instead of just o).

Reply:
Fixed 79e5ac2f2d857af41bb79c097ead99919f7c6e90